### ISSUE TITLE
--testnet command copies over extra fields

### ISSUE THREAD

**Issue by 3xtr4t3rr3str14l** [ChainSafe/lodestar#1266]
**Describe the bug**

When running `yarn run cli beacon init --testnet altona`, the resultant beacon.config.json also contains these parameters:
```
  "templateConfigFile": null,
  "testnet": "altona"
```

**Expected behavior**

I would expect to see the config created without those 2 parameters mentioned above.

**Comment by 3xtr4t3rr3str14l** [comment]
Additionally, the following options that are specified in the altona.ts are not copied over to the resultant beacon.config.json when running the same command as above:
```
  "api": {
    "enabled": true,
    "namespaces": ["beacon", "validator"],
    "host": "127.0.0.1",
    "port": 9596,
    "cors": {
      "origin": "*"
    }
  },
...
"bootMultiaddrs": [
      "/ip4/51.15.97.240/tcp/9000/p2p/16Uiu2HAkwVT363kpFmupwJBH5tkhnaNZPQSY7zANnPGB63ikD1Wp",
      "/ip4/51.15.70.7/tcp/9000/p2p/16Uiu2HAmHV1UA1SBnNK7Ztp8ACQ8DzHwNnR49VDEPBavCU33PtVE"
    ],
...
"localMultiaddrs": ["/ip4/0.0.0.0/tcp/30607"]
```

The fix to this issue should also make sure that `localMultiaddrs`, `bootMultiaddrs`, and `api` are copied over to the beacon.config.json

**Comment by wemeetagain** [comment]
resolved in #1301
